src/ui/control/fullscreen_control.js

```diff
@@ -94,11 +94,19 @@ class FullscreenControl {
     }
 
     _updateTitle() {
-        const title = this._isFullscreen() ? "Exit fullscreen" : "Enter fullscreen";
+        const title = this._getTitle();
         this._fullscreenButton.setAttribute("aria-label", title);
         this._fullscreenButton.title = title;
     }
 
+    _getTitle() {
+        if (this._isFullscreen()) {
+            return this._map._translateUiString("FullscreenControl.ExitTooltip") || "Exit fullscreen";
```

Comment:
Ultimately I think we should follow what I understand to be the [general best practice](https://docs.microsoft.com/en-us/dotnet/standard/globalization-localization/globalization#use-resource-files) of pulling all the default (en) strings out of the code and into an `en` resource module analogous to the `uk` one you've created. Then `._translateUiString()` would become something more like `._getUiString()` and would be responsible for returning the default-or-localized version.